### DISCUSSION TITLE
Does this plugin support iCUE LINK H100i RGB AIO Liquid CPU Cooler with QX fans and icue link system?

### DISCUSSION THREAD

**Discussion by nobleflame** [EvanMulawski/FanControl.CorsairLink#113] [Q&A]
Hi! I've got a new PC with the above cooler and 6 corsair fans hooked up via the icue link system. 

I can't use icue because it's ruining my fps in a lot of games. 

I've tried the plugin, but all I'm getting is the "CorsairLink could not initialize or has no sensors" error. 

Any idea what I can do?

**Comment by EvanMulawski** [comment]
iCUE Link devices are not yet supported (see #111 and #86).

[Device Support](https://github.com/EvanMulawski/FanControl.CorsairLink#device-support)

**Reply by nobleflame** [reply]
Ahh thanks for the response. You’ve saved me some time.

Any idea if / when it’ll be supported in the future?

cheers for all of your work :)

**Reply by EvanMulawski** [reply]
@nobleflame This is now supported as of v1.5.0-beta.1

**Reply by nobleflame** [reply]
@EvanMulawski Thanks, mate :)
